### ISSUE TITLE
No native libs in 2.0.49 and 2.0.50 for BoringSSL Static

### ISSUE THREAD

**Issue by zapek** [netty/netty-tcnative#707]
Look at the file size of the jar:
https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static/2.0.48.Final (4.7 MB)
https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static/2.0.49.Final (7 KB)
https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static/2.0.50.Final (3 KB)

**Comment by normanmaurer** [comment]
@zapek this is expected... see https://github.com/netty/netty-tcnative/pull/696

**Comment by zapek** [comment]
Thanks for your reply. I managed to get it to work but I'm a bit puzzled to know if I'm doing it right.

The documentation at https://netty.io/wiki/forked-tomcat-native.html states (for netty-tcnative-boringssl-static)

> This is an uber jar that contains the statically linked libraries for all supported netty-tcnative-boringssl-static-{os_arch}. While this tends to be a fairly large jar, it greatly simplifies the getting started experience since an application need not be concerned with obtaining the correct jar for the platform.

Which is what I had in my build.gradle:
`implementation 'io.netty:netty-tcnative-boringssl-static:2.0.48.Final'`

If I switch to:
`implementation 'io.netty:netty-tcnative-boringssl-static:2.0.50.Final'`

Then my app crashes on startup because it cannot find the bundled library.

I got it to work with:
`implementation 'io.netty:netty-tcnative-boringssl-static:2.0.50.Final:windows-x86_64'`

is this correct? Do I need to specify it for each architectures?
